app/pages/select/controller.js

```diff
@@ -0,0 +1,35 @@
+import { getRelatedOrganisations } from '../../helpers/api/oapi/getOrganisation';
+
+export const getProxyOrganisations = async ({ accessToken, orgId }) => {
+  const organisationsList = await getRelatedOrganisations({ accessToken, orgId });
+  return organisationsList;
+};
+
+export const getSelectContext = async ({ accessToken, orgId, orgName }) => {
+  const organisationsList = await getProxyOrganisations({ accessToken, orgId });
+
+  let radioList;
+
+  if (organisationsList) {
+    radioList = organisationsList.map((org) => ({ value: org.organisationId, text: org.name }));
+    radioList.sort((a, b) => a.text.localeCompare(b.text));
+  }
+
+  const context = {
+    primaryName: orgName,
+    organisationList: radioList,
+  };
+
+  return context;
+};
+
+export const getIsUserProxy = async ({ accessToken, orgId }) => {
+  if (!accessToken) {
+    return false;
+  }
+
+  const proxyOrganisations = await getProxyOrganisations({ accessToken, orgId });
+
+  const userIsProxy = proxyOrganisations.length > 0;
+  return userIsProxy;
+};
```

Comment:
I'm not sure this belongs in a controller. I think it might be better being moved out to a helper.

Also, why make an extra API call? Can't we test if a user can act as a proxy by checking the user object?

Reply:
What attribute of the user object has the setting that lets us know it is a proxy?

Reply:
The changes @leerogers-bjss is working on will add a `relatedOrganisationId` claim to the user if their primary organization has any related organizations. (It's essentially the same I what I referred to as `secondaryOrganisationId`.

This claim will only be present if related organizations exist, so I think we should be able to test for `req.user.relatedOrganisationId`.

Reply:
This will need to be a separate ticket and we will implement that change when it has been finished

Reply:
Why does it need a new ticket?

Reply:
This PR will have to sit open for x number of days until the change has been made.  We put in dummy values while we wait for api work to be finished, then we have a ticket to update the dummy values with real values when the api has been approved.  Otherwise we have a blocker for this PR

Reply:
The API changes are in PR and approved. However, even if the API changes weren't quite ready this could still go in (with the suggested change to check the claims) without any need for it to sit open.

Reply:
please link the pr so i can look at the story are carry on with this ticket

Reply:
https://github.com/nhs-digital-gp-it-futures/BuyingCatalogueIdentity/pull/314